test/known_issues/test-vm-inherited_properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+// Ref: https://github.com/nodejs/node/issues/5350
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+const base = {
+  prop_base: 1,
+};
+
+const sandbox = Object.create(base, {
+  prop_sandbox: {value: 3}
+});
+
+const context = vm.createContext(sandbox);
+
+const result = vm.runInContext("this.hasOwnProperty('prop_base');", context);
```

Comment:
Oh uh... good question. We don't use Standard though (for legacy reasons).

@Trott thoughts? I'd prefer template strings tbh but I think our linter may warn about that currently...